pkgs/tools/package-management/apx/default.nix

```diff
@@ -30,25 +30,24 @@ buildGoModule rec {
   postInstall = ''
     mkdir -p $out/etc/apx
 
-    cat > "$out/etc/apx/config.json" <<EOF
+    # https://github.com/Vanilla-OS/apx/blob/main/config/apx.json
+    cat > "$out/etc/apx/apx.json" <<EOF
     {
-      "containername": "apx_managed",
-      "image": "docker.io/library/ubuntu",
-      "pkgmanager": "apt",
-      "distroboxpath": "${distrobox}/bin/distrobox"
+      "apxPath": "$out/bin/apx",
+      "distroboxpath": "${distrobox}/bin/distrobox",
+      "storageDriver": "btrfs"
     }
     EOF
```

Comment:
Can we just use `substituteInPlace` to modify the upstream configuration file, saving the trouble to update this everytime.